radioapp/ui/ui_manager.py

```diff
@@ -0,0 +1,41 @@
+import tkinter
+from tkinter import ttk
+from .home import UI_Home
+from .settings import UI_Settings
+
+class UI():
+
+    #Holds the main window
+    def __init__(self):
+        #initialize window and frame
+        self.window = tkinter.Tk()
+        self.window.columnconfigure(0, weight=1, uniform=1)
+        self.window.rowconfigure(0, weight=1, uniform=1)
+        self.window.geometry("160x128")
+        self.window.resizable(0,0)
+        self.window.pack_propagate(0)
+        
+        #Set defaults for the tkinter instance
+        self.style = ttk.Style(self.window)
+        self.style.configure('TFrame', background='blue')
+        self.window.option_add("*Background", "blue")
+        self.window.option_add("*Foreground", "white")
+
+
+        #initialize the different windows
+
+        #initalize to the homescreen
+        self.goHome()
+        self.window.mainloop()
+
+    def goHome(self):
+        UI_Home(self.window, self.goSettings, None, None)
```

Comment:
For side effects, the constructor is just setting up the view. I think that is pretty normal, they don't actually change the main window UI, just config for their own UI frame.

Reply:
I could possibly move the adding of the frame to the window outside of the constructor, but I would need to consult tkinter, and since our project is so simple may just cause redundancy